openapi/types.rs

```diff
@@ -0,0 +1,138 @@
+/// Types used for REST communication with the svc-cargo server
+
+use serde::{Deserialize, Serialize};
+use utoipa::{IntoParams, ToSchema};
+use chrono::NaiveDateTime;
+
+/// API Endpoint to Cancel a Flight
+pub const ENDPOINT_CANCEL: &str = "/cargo/cancel";
+
+/// API Endpoint to Confirm a Flight
+pub const ENDPOINT_CONFIRM: &str = "/cargo/confirm";
+
+/// API Endpoint to Query for Available Flights
+pub const ENDPOINT_QUERY: &str = "/cargo/query";
+
+/// API Endpoint to Get Vertiports for a Region
+pub const ENDPOINT_REGION: &str = "/cargo/region";
+
+/// Request Body Information for Flight Query
+#[allow(dead_code)]
+#[derive(Debug, Clone, IntoParams, ToSchema)]
+#[derive(Deserialize, Serialize)]
+pub struct FlightQuery {
+    vport_depart_id: String,
+    vport_arrive_id: String,
+    timestamp: NaiveDateTime,
+    weight_kg: f32,
```

Comment:
Maybe rename `timestamp` or adding some docs about what it stands for. Does it mean the current time at which a flightquery is made, or that a customer wants a flight at a particular time?